metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/configuration/MetadataDatabaseConfiguration.java

```diff
@@ -45,6 +46,7 @@
         basePackages = "uk.ac.ebi.ampt2d.metadata.persistence.repositories",
         transactionManagerRef = "metadataTransactionManager")
 @EnableJpaAuditing
+@EnableRetry
```

Comment:
Can we move @EnableRetry to MetadataMainApplicationConfiguration because that config class is better in sense of this annotation?